i18n/french.js

```diff
@@ -96,7 +96,9 @@ module.exports = {
                 file: "fichier",
                 errorMsg: "valeur incriminée",
                 position: "position",
-                inspect: "inspecter"
+                inspect: "inspecter",
+                disable_scarf: "Cette dépendence peut recolter des données contre votre volonté, pensez donc à la désactiver en fournissant la variable d'environnement SCARF_ANALYTICS",
+                keylogging: "Cette dépendence peut obtenir vos entrez clavier ou de souris. Cette dépendence peut être utiliser en tant que 'keylogging' attacks/malwares."
```

Comment:
```suggestion
                disable_scarf: "Cette dépendance peut récolter des données contre votre volonté, pensez donc à la désactiver en fournissant la variable d'environnement SCARF_ANALYTICS",
                keylogging: "Cette dépendance peut obtenir vos entrées clavier ou de souris. Cette dépendance peut être utilisée en tant que 'keylogging' attacks/malwares."
```

Reply:
Thanks for your return, I provide a fix in the next commit